File: src/main/java/no/nav/melosys/eessi/models/sed/medlemskap/impl/VedtakA002.java
package no.nav.melosys.eessi.models.sed.medlemskap.impl;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;
import no.nav.melosys.eessi.models.sed.nav.Periode;

@Data
public class VedtakA002 {

    private Periode annenperiode;
    private String begrunnelse;
    @JsonProperty("id")
    private String resultat;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/medlemskap/impl/VedtakA002.kt
package no.nav.melosys.eessi.models.sed.medlemskap.impl

import no.nav.melosys.eessi.models.sed.nav.Periode
import com.fasterxml.jackson.annotation.JsonProperty

data class VedtakA002(
    var annenperiode: Periode? = null,
    var begrunnelse: String? = null,
    @JsonProperty("id")
    var resultat: String? = null
)
